pkg/cmd/hoptimize.go

```diff
@@ -95,9 +95,7 @@ var hoptimizeCmd = &cobra.Command{
 		defer cancel()
 
 		go func() {
-			c := make(chan os.Signal)
-			signal.Notify(c, syscall.SIGINT, syscall.SIGTERM)
-			<-c
+			signal.Notify(make(chan os.Signal), syscall.SIGINT, syscall.SIGTERM)
```

Comment:
but we need to block the call

the example did not change https://pkg.go.dev/os/signal#Notify

Reply:
modified according to the requirement of blocking the call.
please check again.